<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemSettingMainWindow</class>
 <widget class="QMainWindow" name="SystemSettingMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_trading">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_accounts">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>92</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_password">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>92</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_client">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>92</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_setting">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>611</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton_confirm">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>400</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>400</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_confirm_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>400</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>应用</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>密码修改</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_newpassword">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>输入新密码</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_confirm">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>确认新密码</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_rule">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>531</width>
        <height>101</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>提示：为提升密码安全系数，新密码位数至少8位，且需包含下列字符中的至少三项：数字、小写字母、大写字母、特殊字母：！@#$%*&amp;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_oldpassword">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>输入原密码</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>请上传日志</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_Menu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>181</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>交易</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>账号</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="pushButton_changepassword">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>修改密码</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_logUpload">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>日志上传</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_clientlock">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>客户端锁定</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_margintradesetting">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>预留信息设置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_about">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>关于</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
